src/main/java/uk/gov/justice/digital/delius/controller/secure/AuthenticationController.java

```diff
@@ -0,0 +1,118 @@
+                    @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class)
+            })
+    @GetMapping("/authenticate")
+    public ResponseEntity authenticate(@ApiParam(name = "username", value = "LDAP username", example = "TESTUSERNPS", required = true) final @RequestParam String username,
+                                       @ApiParam(name = "password", value = "LDAP password", example = "hello123456", required = true) final @RequestParam String password) {
+        boolean authenticated = userService.authenticateUser(username, password);
+        if(!authenticated) {
+            return new ResponseEntity(HttpStatus.UNAUTHORIZED);
+        }
+        return new ResponseEntity(HttpStatus.OK);
+    }
+
+    @ApiOperation(
+            value = "Find user details of a user held in Delius Identity (LDAP)",
+            authorizations = {@Authorization("ROLE_AUTH_DELIUS_LDAP")})
+    @ApiResponses(
+            value = {
+                    @ApiResponse(code = 200, message = "OK", response = UserDetails.class),
+                    @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+                    @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
+                    @ApiResponse(code = 404, message = "Not Found", response = ErrorResponse.class)
+            })
+    @RequestMapping(value = "/users/{username}/details", method = RequestMethod.GET)
+    public ResponseEntity<UserDetails> findUser(@ApiParam(name = "username", value = "LDAP username", example = "TESTUSERNPS", required = true)  final @PathVariable("username") String username) {
+        return userService.getUserDetails(username).map(userDetails -> new ResponseEntity<>(userDetails, OK))
+                .orElse(new ResponseEntity<>(NOT_FOUND));
+    }
+
+    @ApiOperation(
+            value = "Change password a users (LDAP) account",
+            authorizations = {@Authorization("ROLE_AUTH_DELIUS_LDAP")})
+    @ApiResponses(
+            value = {
+                    @ApiResponse(code = 200, message = "Password Changed"),
+                    @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+                    @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
+                    @ApiResponse(code = 404, message = "Not Found", response = ErrorResponse.class)
+            })
+    @RequestMapping(value = "/users/{username}/password", method = RequestMethod.POST)
+    public ResponseEntity changePassword(@ApiParam(name = "username", value = "LDAP username", example = "TESTUSERNPS", required = true) final @PathVariable("username") String username,
+                                         @ApiParam(name = "password", value = "LDAP new password", example = "newpassword123", required = true) final @RequestParam("password") String password) {
```

Comment:
same issue here with password as a request parameter

Reply:
Agreed was confused why it was done that way